src/xml/attributes.cr

```diff
@@ -29,6 +30,10 @@ struct XML::Attributes
     raise IndexError.new
   end
 
+  def size
```

Comment:
Can we remove this?

Reply:
I think the idea was that `XML` having a `size` method is good. The reason that this *appears* here is so that `XML#size` does not show deprecation.

Maybe right now XML's implementation for `size` is not amazing but that doesn't mean it is not useful.

Reply:
(But I'm not sure, maybe it's *actually* not useful, in that case nevermind)

Reply:
The question whether `#size` is useful for `XML::Attributes` is a different one from generic `Enumerable#size`. This is the minimal change to remove `Enumerable#size` without disturbing `XML::Attributes`. As mentioned above, I'd rather have a separate discussion for that.

Reply:
Basically, adding this piece of code is a smaller change than not adding it would be